tensorflow/python/ops/nn_impl.py

```diff
@@ -856,9 +856,7 @@ def depthwise_conv2d_v2(input,
     filter: 4-D with shape
       `[filter_height, filter_width, in_channels, channel_multiplier]`.
     strides: 1-D of size 4.  The stride of the sliding window for each
-      dimension of `input`. Must have `strides[0] = strides[3] = 1`, and for
```

Comment:
You'll need to re-add something here, but you need to explicitly mention that _only_ equal strides are supported (I'm guessing only if `strides[0]` and `strides[3]` are 1?).  Do some testing to confirm, and fix where it's mentioned above as well.